_rules/audio-transcript-2eb176.md

```diff
@@ -22,27 +25,29 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
+This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element for which at least one of the following is true:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- the element has an `autoplay` [attribute value][] of `true`; or
+- the element has a [play button][] that is [visible][].
```

Comment:
why was this change made?

Reply:
Because from the survey discussion we agreed that to test the rule 1.2.1 the AT part is for a different checkpoint so it was made as an assumption instead: https://docs.google.com/spreadsheets/d/1OSkPFocXk4K3zYLnwS78WLsWO4PvE5yRcsauyefuIUI/edit#gid=1605073031